### ISSUE TITLE
Add docs for how to use tracing libraries in the openzipkin org

### ISSUE THREAD

**Issue by adriancole** [openzipkin/openzipkin.github.io#33]
Probably make sense for us to host basic how-to for the tracers that we host. For example, the ruby tracer and brave.

Particularly, @virtuald requested docs be made easier to find for brave.

**Comment by abesto** [comment]
I'd expect to find detailed documentation for each tracer in the README / documentation of those individual tracers. In systems with lots of coexisting components (plugins etc) I like having a central location with high-level / getting started documentation I can just click through, but in this case each time I use the documentation, I'm looking for the documentation of one very specific library.

Long story short: maybe this is about improving / standardizing documentation of tracers?

**Comment by adriancole** [comment]
Could be. Fair to wait before taking more action
